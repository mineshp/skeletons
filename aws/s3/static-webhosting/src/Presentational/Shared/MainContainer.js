import React from 'react';
import { Switch, Route } from 'react-router-dom';
import { WelcomeConnectedComponent } from '../../Container/Welcome';

import NotFound from './NotFound';

const Main = () => (
  <Switch>
    <Route exact path="/" component={WelcomeConnectedComponent} />
    <Route path="*" component={NotFound} />
  </Switch>
);

export default Main;
