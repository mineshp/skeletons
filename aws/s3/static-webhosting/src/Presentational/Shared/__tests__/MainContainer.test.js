import React from 'react';
import { shallow } from 'enzyme';
import MainContainer from '../MainContainer';

describe('Presentational > MainContainer', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(
      <MainContainer />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
