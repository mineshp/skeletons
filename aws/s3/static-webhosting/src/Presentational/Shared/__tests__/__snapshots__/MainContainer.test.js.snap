// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Presentational > MainContainer Renders correctly 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    path="*"
  />
</Switch>
`;
